homework/english/skiba_olga/lecture2/grammar2.md

```diff
@@ -0,0 +1,38 @@
+# Упражнения Past Simple и Present Perfect
+
+### Упражнение 1.  Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+1. I've just had a nice pot of coffee. Would you like a cup?
+2. I've not seen Steve this morning yet.
+3. Carol and I are old friends. I've known her since I was a freshman in high school.
+4. Maria has had a lot of problems since she come to this country.
+5. I went to Paris in 2003 and 2006.
+6. A car came round the corner and I jumped out of the way.
+7. Don’t throw the paper away because I've not read it yet.
+8. Is Jim going to eat lunch with us today? — No. He has eaten already. He aet lunch an hour ago.
+9.  Since we've started doing this exercise, we completed  some sentences.
+10. I've never been to Italy.
+
+### Упражнение 2. Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+1. Have you ever worked in a shop?
+2. I worked at my uncle’s shop when I was younger.
+3. It’s the first time I've been on a ship.
+4. Ann is looking for her key. She can’t find it. She lost her key.
+5. How many symphonies Beethoven has composed?
+6. Look! Somebody spilt ink on the notebook.
```

Comment:
somebody has spilled ink on the notebook.

Reply:
I changed it to present perfect, but the 3rd form of the verb spill is spilt.